backend/controllers/CommentsController.ts

```diff
@@ -30,6 +30,14 @@ const CommentsController = {
     } catch (err: any) {
       return res.status(500).json({ error: req.t('server_error'), err });
     }
+  },
+  async deleteLocationComments(req: Request, res: Response) {
+    try {
+      await Comment.deleteOne({ _id: req.params.id });
+      return res.status(200).json({});
```

Comment:
why empty object ? should add req.t

Reply:
i removed empty object and now i return just this res.status(200).json(), because i deleting comment and i don't need data